### PR TITLE
ROX-24581 : Changes to add publish date to  imageCVEs query

### PR DESCRIPTION
**Pull request by ksurabhi91**
This PR adds publish date field to imageCVEs query so UI can display  publish date in Workload CVEs table
Testing:
Made get query in graphiql to test the query response. see screenshot attached
Screenshot 2024-09-19 at 11 03 26 AM

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [ ] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

change me!

### CODE REVIEW

**Review by charmik-redhat — APPROVED** [review]
🚢

File: central/graphql/resolvers/image_cve_core.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ func init() {
 				"exceptionCount(requestStatus: [String]): Int!",
 				"images(pagination: Pagination): [Image!]!",
 				"topCVSS: Float!",
+				"cvePublishedDate: Time",
```
Comment (vjwilson) [inline central/graphql/resolvers/image_cve_core.go:43]: Just want to make sure this is the same key that appears in the other GraphQL queries, so the UI can re-use code if necessary?

**Review by vjwilson — COMMENTED** [review]
Just one question

File: central/graphql/resolvers/image_cve_core.go (hunk 1 above)
Comment (ksurabhi91) [reply central/graphql/resolvers/image_cve_core.go:43]: key in other query is publishedOn. but it is a different query so not sure if code can be re used

Comment (vjwilson) [reply central/graphql/resolvers/image_cve_core.go:43]: I don't mean reusing the query code; I mean the components that take the query result. What is the reason to call the data `cvePublishedDate` here, if the same data is called `publishedOn` in another resolver?

Comment (ksurabhi91) [reply central/graphql/resolvers/image_cve_core.go:43]: ok got it. i can change it to publishedOn

**Pull request merged**